containers/daap-python-base/src/var/task/data_platform_paths.py

```diff
@@ -280,7 +310,7 @@ def metadata_path(self):
         key = os.path.join(
             "metadata",
```

Comment:
This path now doesn't quite align with the prefix in `get_latest_version` where there is no `metadata`